erpnext/templates/pages/rfq.html

```diff
@@ -12,12 +12,6 @@
 	{% include "templates/includes/breadcrumbs.html" %}
 {% endblock %}
 
-{% block style %}
```

Comment:
order.css has been removed, while accessing rfq page system showing an error order.css has not found